node/src/components/storage.rs

```diff
@@ -1474,29 +1557,18 @@ impl Storage {
             .collect()
     }
 
-    /// Get the switch block for a specified era number in a read-only LMDB database transaction.
-    ///
-    /// # Panics
-    ///
-    /// Panics on any IO or db corruption error.
-    pub fn transactional_get_switch_block_by_era_id(
+    /// Get the switch block for a specified era number.
+    pub fn read_switch_block_by_era_num(
```

Comment:
Nitpick: we generally refer to "era IDs" rather than "era numbers"

Reply:
Fixed in f8326e350